apps/web/src/pages/get-started/consts/shared.tsx

```diff
@@ -16,8 +16,26 @@ export const StepButton = styled(Button)`
   display: block;
 `;
 
-export function GetStartedLink({ children, ...linkProps }: React.AnchorHTMLAttributes<HTMLAnchorElement>) {
-  return <StyledLink {...linkProps}>{children}</StyledLink>;
+export function GetStartedLink({
+  children,
+  ...linkProps
+}: React.AnchorHTMLAttributes<HTMLAnchorElement> & {
+  event?: string;
+}) {
+  const segment = useSegment();
+
+  return (
+    <StyledLink
+      {...linkProps}
+      onClick={() => {
+        if (linkProps.event) {
+          segment.track(`${linkProps.event} - [Get Started]`, { href: linkProps.href });
```

Comment:
Love it! :)

I wonder if maybe we should send the channel/action by property so that the consumers of the event could group by the channels easily, what do you think?

Reply:
oh a great suggestion 🥳